holidays/countries/norway.py

```diff
@@ -35,7 +35,8 @@ class Norway(HolidayBase, ChristianHolidays, InternationalHolidays):
 
     country = "NO"
     default_language = "no"
-    supported_languages = ("en_US", "no", "uk")
+    supported_languages = 
+    ("en_US", "no", "th", "uk")
```

Comment:
```suggestion
    supported_languages = ("en_US", "no", "th", "uk")
```
No need to split them into new lines :)